client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -33,15 +33,16 @@ export default React.createClass( {
 
 	render() {
 		const { site, redirect, linkDisplay } = this.props;
-		const buttonElement = linkDisplay ? 'button' : Button;
```

Comment:
Yes, I'd say that's it. A borderless button is essentially a link, but it may be rendered inside some other button-y wrapper making that context sensical. With the 'is-link' we should just use a link. It's probably still in the css because there are a handful of is-link buttons floating around out there.